### PR TITLE
Make the python operator usable in TQL2

### PR DESCRIPTION
**Pull request by tobim**

### CODE REVIEW

File: libtenzir/builtins/operators/python.cpp
Comment (jachris) [inline libtenzir/builtins/operators/python.cpp:1]: I took the liberty to push to this branch (improve error messages and worked around on some limitations of the current implementation, e.g., w.r.t. throwing). 

I hope that's okay. Please resolve if my edits are fine.

**Review by jachris — APPROVED** [review]
Thanks!

File: libtenzir/builtins/operators/python.cpp
Comment (dominiklohmann) [reply libtenzir/builtins/operators/python.cpp:1]: Is it intended that you use emit for one error and throw for the other?

Comment (jachris) [reply libtenzir/builtins/operators/python.cpp:1]: Oops, not at all. Thank you!

**Pull request merged**